jobs/stackdriver-nozzle/templates/stackdriver-nozzle-ctl.erb

```diff
@@ -32,8 +32,8 @@ case $1 in
     echo $$ > $PIDFILE
 
     exec chpst -u vcap:vcap /var/vcap/packages/stackdriver-nozzle/bin/stackdriver-nozzle \
-      >>$LOG_DIR/stackdriver-nozzle.stdout.log \
-      2>>$LOG_DIR/stackdriver-nozzle.stderr.log
+      >> >(tee $LOG_DIR/stackdriver-nozzle.stdout.log) \
```

Comment:
We were thinking that we might want a more general purpose solution, but it depends on whether we want to capture logs from other jobs that are logging to files instead of syslog. It seems like something we probably want to do, but I'm not sure.